W3ChampionsStatisticService/Services/PlayerAkaProvider.cs

```diff
@@ -11,40 +11,37 @@ namespace W3ChampionsStatisticService.Services
 {
     public class PlayerAkaProvider
     {
-        private static CachedData<List<PlayerAka>> PlayersAkaCache = new CachedData<List<PlayerAka>>(() => FetchAkasSync(), TimeSpan.FromMinutes(60));
-        
-        public static List<PlayerAka> FetchAkasSync()
+        private readonly ICacheData<List<PlayerAka>> _userAccountsCache;
+
+        public PlayerAkaProvider(ICacheData<List<PlayerAka>> userAccountsCache)
         {
-            try
-            {
-               return FetchAkas().GetAwaiter().GetResult();
-            }
-            catch
-            {
-                return new List<PlayerAka>();
-            }
+            _userAccountsCache = userAccountsCache;
         }
 
-        public static async Task<List<PlayerAka>> FetchAkas() // list of all Akas requested from W3info
-        { 
+        public static async Task<List<PlayerAka>> FetchBattleNetAccounts() // list of all Akas requested from W3info
```

Comment:
This function isn't getting bnet accounts, it's getting a human-readable reference for us to consult bnet accounts against.

Reply:
Renamed, tried to make it readable but didn't understand currently)